core/pkg/server/sender.go

```diff
@@ -544,6 +544,7 @@ func (s *Sender) sendRequestDefer(request *spb.DeferRequest) {
 		// manager.
 		s.fileWatcher.Finish()
 		if s.fileTransferManager != nil {
+			s.runfilesUploader.UploadRemaining()
```

Comment:
just making sure that the file records are saved in the transaction log for offline mode

Reply:
ok i think it is fine

Reply:
yep! this doesn't generate any records; the corresponding records have already been processed